core-bundle/public/limit-height.js

```diff
@@ -18,21 +18,18 @@ window.addEventListener('DOMContentLoaded', function () {
         // Return if there is no height value
         if (!hgt) return;
 
-        const toggler = document.createElement('div');
-        toggler.classList.add('limit_toggler');
+        const height = div.offsetHeight;
+
+        // Resize the element
+        div.style.height = hgt+'px';
```

Comment:
should this be skipped as well if the element is high enough?

Reply:
No, if an element has `.h40`, it shall be 40px high. And if its content is higher, a toggle shall be added additionally.

Reply:
### Before

<img width="769" alt="" src="https://user-images.githubusercontent.com/1192057/185409671-2b48f9f1-24cb-46ac-893b-34990256cf2d.png">

### After

<img width="698" alt="" src="https://user-images.githubusercontent.com/1192057/185409697-aef81be6-1a4b-432e-a9ae-f242831ca6f5.png">

Reply:
Strange, but I see that was already the case currently.